interfaces/cython/SConscript

```diff
@@ -109,8 +109,8 @@ elif localenv['python_prefix']:
     elif localenv['libdirname'] != 'lib':
         # 64-bit RHEL / Fedora etc. or e.g. x32 Gentoo profile
         extra = localenv.subst(
-            ' --prefix=${python_prefix}'
-            ' --install-lib=${{python_prefix}}/${{libdirname}}/python{}/site-packages'.format(py_version))
```

Comment:
The addition of the extra set of parentheses around the whole string should not be necessary.

Reply:
But without parentheses there will be KeyError as .format() method is applied after concatenation and encounter the {} expression. I wonder why the previous time I tested early patch didn't result the issue observed.

The KeyError points to next string so initially I was confused what is happened.

Do you mean I need remove [SCons] from pr title?

The easy way to reproduce this problem if assumed we want print `{test_1} {test_2} test_value` is run
```
value = 'test_value'
print(
    '{test_1}
    ' {{test_2}} {}'.format(value)
    )
```
KeyError pointed to second string of print() appeared. While
```
value = 'test_value'
print(
    ('{{test_1}}
    ' {{test_2}} {}').format(value)
    )
```
prints the desired string.

Reply:
@speth is referring to the extra set of round brackets `()` that you added. The difference between your two code samples is due to the extra set of curly brackets `{}` that you added around `test_1`. For instance, this works, without the extra parentheses:
```python
>>> print(
... '{{test_1}}'
... ' {{test_2}} {}'.format(value)
... )
{test_1} {test_2} test_value
```

Reply:
Yes, I unferstanded.  I just tried to be too careful here. I'll remove parenthesis.